src/commands/hosting-channel-deploy.ts

```diff
@@ -151,7 +131,28 @@ export default new Command("hosting:channel:deploy [channelId]")
 
       logger.info();
       const deploys: { [key: string]: ChannelInfo } = {};
-      sites.forEach((d) => {
+      for (const d of sites) {
+        try {
+          await addAuthDomain(projectId, d.url);
```

Comment:
Would it behoove us to write an `addAuthDomains` function that allowed us to add more than one at once? Then we would only have to make one update of the list.